packages/maskbook/src/plugins/Wallet/UI/ImportWalletDialog/FromJson.tsx

```diff
@@ -0,0 +1,5 @@
+import type { FC } from 'react'
+
+export const FromJson: FC = () => {
+    return <div>from json</div>
```

Comment:
Emm，Why is such a component needed, as this now seems pointless

Reply:
It's a placeholder. Should integrate the ImportFromJson component

Reply:
So this is a WIP?